src/v/kafka/server/handlers/delete_records.cc

```diff
@@ -145,6 +145,11 @@ static ss::future<result_t> prefix_truncate(
             kerr = error_code::request_timed_out;
         } else if (errc == raft::errc::invalid_truncation_offset) {
             kerr = error_code::offset_out_of_range;
+        } else if (errc == cluster::errc::feature_disabled) {
+            vlog(
+              klog.warn,
+              "Prefix truncate request rejected - feature not yet activated");
```

Comment:
I don't think this message would make sense to a user: should say "DeleteRecords" so that they know what kind of external request this is talking about.